onmt/translate/__init__.py

```diff
@@ -1,4 +1,5 @@
 from onmt.translate.Translator import Translator
+from onmt.translate.Translation import Translation
 from onmt.translate.Beam import Beam, GNMTGlobalScorer
 
-__all__ = [Translator, Beam, GNMTGlobalScorer]
+__all__ = [Translator, Translation, Beam, GNMTGlobalScorer]
```

Comment:
May I ask what is the difference of the `Translation` and `Translator`?  Can we unify them?  Besides, while at it, do we want to change all CamelCase to hyphen_lowercase?

Reply:
Oh, I purposely split them. Translator does the translating. Translation is the class that holds the translation (and all its parts: nbest, attention, etc.) 

Yeah, we can do that. What is the style again? All variables should be underscore_lowercase?